playwright-e2e/pages/page-base.ts

```diff
@@ -166,14 +166,20 @@ export default abstract class PageBase implements PageInterface {
       street = '415 MAIN ST',
       city = 'CAMBRIDGE',
       zipCode = '02142',
-      telephone = generateRandomPhoneNum(),
-      phoneLabel = 'Telephone Contact Number'
+      telephone = generateRandomPhoneNum()
     } = opts;
 
-    const mailAddressForm = new Address(this.page, { label: new RegExp(/contact information|Mailing Address/) });
+    var labels;
+    if (!opts.labels) {
+      labels = { phone: 'Phone',country: 'Country',state:'State',zip:'Zip Code', city:'City'}
+    } else {
+      labels = opts.labels;
+    }
+
+    const mailAddressForm = new Address(this.page, { label: new RegExp(/Your contact information|Contact Information|Mailing Address/) });
```

Comment:
should "Your" be included in the the RegExp? There might be a chance where the text "Your child"/"Your child's" is encountered.

Reply:
So far this seems to be passing for me locally on other tests (in particular singular), so I'm inclined to leave this one alone...but if it breaks on circleCI, this will be my first suspect!